### DISCUSSION TITLE
My question is related to agame.com (Details inside)

### DISCUSSION THREAD

**Discussion by NomanBaigA** [aryamangodara/Arcade-Plaza#12] [Q&A] [answered]
you are using agame.com for games. is agame allows us to create apps by using their website? or do we need any permission to do that?

the app is amazing and already gives 5-star rating for that.

**Comment by aryamangodara** [comment, marked as answer]
I also do not know it 😅. As app is not very famous I guess we might never get caught 😶😂.

I am very delighted you loved my app.

Also please give new ideas or do some contribution to make this app better!
